style/js/shield_defs.js

```diff
@@ -168,6 +168,28 @@ export function loadShields(shieldImages) {
     },
   };
 
+  var padding_ca_nb = {
+    left: 4,
+    right: 4,
+    top: 5,
+    bottom: 5,
+  };
+  shields["CA:NB:primary"] = {
+    backgroundImage: shieldImages.shield40_ca_nb_primary,
+    textColor: "black",
+    padding: padding_ca_nb,
+  };
+  shields["CA:NB:secondary"] = {
+    backgroundImage: shieldImages.shield40_ca_nb_secondary,
```

Comment:
I think the `colorLighten` option would allow us to consolidate these three images into one:

https://github.com/ZeLonewolf/openstreetmap-americana/blob/70db71e788467bb452f4ce68ea3e88e9c3129a6d/style/js/shield_defs.js#L285

Reply:
Does that apply to the text as well, or only the background image?

Reply:
Not sure; haven’t tried yet. Here’s the documentation for it:

https://github.com/ZeLonewolf/openstreetmap-americana/blob/70db71e788467bb452f4ce68ea3e88e9c3129a6d/style/js/shield_defs.js#L1594-L1597 https://github.com/ZeLonewolf/openstreetmap-americana/blob/70db71e788467bb452f4ce68ea3e88e9c3129a6d/style/js/shield.js#L241-L243

If it doesn’t keep the text black, then the approach you took is perfectly reasonable.

Reply:
The current implementation applies the recoloration to both the text and shield.  In retrospect this is a bug because text color is separately configured.

Reply:
On second thought, it's not _necessarily_ a bug because we rely on this behavior for the Georgia special case shields 🤔

Reply:
I depended on it for the Yukon shields too :grimacing: 

I wish there were an easy way to palette-swap SVGs with arbitrary fill and stroke. That would have made the extra "home down" plates introduced in #271 unnecessary.

Reply:
OK, it’s not a big deal to me if we can’t take advantage of this optimization here.